File: app/src/main/java/mo/wall/org/dialog/MyDialogContract.java
package mo.wall.org.dialog;

import org.wall.mo.base.mvp.BaseContract;
import org.wall.mo.base.mvp.BaseContract.BasePresenter;

public class MyDialogContract {

    public interface View extends BaseContract.BaseView {
    }

    public static abstract class Presenter extends BaseContract.BasePresenter<View> {
    }
}

File: app/src/main/java/mo/wall/org/dialog/MyDialogContract.kt
package mo.wall.org.dialog

import org.wall.mo.base.mvp.BaseContract
import org.wall.mo.base.mvp.BaseContract.BasePresenter

class MyDialogContract {

    interface View : BaseContract.BaseView

    abstract class Presenter : BaseContract.BasePresenter<View>()
}
